### DISCUSSION TITLE
Preview broken

### DISCUSSION THREAD

**Discussion by valentin78** [usebruno/bruno#1489] [General]
update version from 1.4.0 to 1.7.1 and looks like you lost ability to preview (PDF). It was working in 1.4.0 :
![image](https://github.com/usebruno/bruno/assets/4290501/add9166b-77c3-4d83-800a-ef39554c0524)
Also if I try to save response (application/pdf) to file, I'm getting error message
![image](https://github.com/usebruno/bruno/assets/4290501/db1a4e91-8ee6-4850-9b30-d40ae594868e)

**Comment by helloanoop** [comment]
Hi @valentin78 

This has been fixed in #1490 and I will release a patch fix for this shortly.
